lib/mailroom/imap.ex

```diff
@@ -143,6 +143,14 @@ defmodule Mailroom.IMAP do
     end
   end
 
+  def uid_fetch(pid, number_or_range, items_list, opts \\ []) do
```

Comment:
Please add a test for this. If you start an IMAP session with a server you control with `debug: true` you can get the server output and then, removing sensitive information, create a test like the others in imap_test.exs

Reply:
done